web-ng/src/app/components/feature-panel/feature-panel.component.ts

```diff
@@ -79,6 +80,34 @@ export class FeaturePanelComponent implements OnInit, OnDestroy {
             .pipe(map(feature => project.layers.get(feature.layerId)!))
         )
       );
+    this.photoUrls = new Map();
+    this.observations$.forEach((observations) => {
```

Comment:
I've never seen Observable used this way before, interesting!

Reply:
from the behaviour it seems that it creates subscription (?)